.github/workflows/build.yml

```diff
@@ -1,29 +1,32 @@
 name: Build
 
-on: 
-  pull_request:
-    branches:
-      - master
+on:
+    pull_request:
+        branches:
+            - master
 
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        node: [10.x]
-        os: [ubuntu-latest]
+    build:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                node: [10.x]
+                os: [ubuntu-latest]
 
-    steps:
-      - name: Set up Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node }}
-      - name: Downloads Checkouts...
-        uses: actions/checkout@v1
-      - name: Portpolio Build...  
-        run: |
-          npm install
-          cp -rf config/webpack.config.dev.js node_modules/react-scripts/config/webpack.config.dev.js
-          npm run build-css
-          npm run build --if-present
-        
\ No newline at end of file
+        steps:
+            - name: Set up Node.js ${{ matrix.node }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node }}
+            - name: Downloads Checkouts...
+              uses: actions/checkout@v1
+            - name: Create Firebase Configuration for Neon K.I.D
+              env:
+                  FIREBASE_CONFIG: ${{ secrets.FIREBASE_CONFIG }}
+              run: |
+                  echo $FIREBASE_CONFIG > src/components/Server/config.json
```

Comment:
Check current path,,

Reply:
This problem is not path problem,
My commit does not exist server code for firebase..